test/system/src/test/java/io/pravega/test/system/MultiReaderWriterWithFailOverTest.java

```diff
@@ -289,21 +287,21 @@ public void multiReaderWriterWithFailOverTest() throws Exception {
             //delete readergroup
             log.info("Deleting readergroup {}", readerGroupName);
             readerGroupManager.deleteReaderGroup(readerGroupName);
+            //seal the stream
```

Comment:
no, `multiReaderTxnWriterWithFailOverTest` does not have the same issue since the `ControllerImpl` uses a separate executor. Updated the code to use external executor for controller client.